### PR TITLE
Remove variables from log messages

### PR DESCRIPTION
**Pull request by vogdb**
Depends on #220

### CODE REVIEW

File: lib/cm-api-client.js [hunk 1]
```diff
@@ -44,10 +45,10 @@ CMApiClient.prototype._request = function(action, data) {
     .then(function(response) {
       var error = response['error'];
       if (error) {
-        serviceLocator.get('logger').error('cm-api', 'response', response);
+        serviceLocator.get('logger').error('cm-api response', response);
```
Comment (tomaszdurka) [inline lib/cm-api-client.js]: Let's wrap this into a hash.

File: lib/cm-api-client.js [hunk 2]
```diff
@@ -44,10 +45,10 @@ CMApiClient.prototype._request = function(action, data) {
...
         throw new JanusError.CmApi(error['type'] + '. ' + error['msg']);
       }
-      serviceLocator.get('logger').info('cm-api', 'response', response.body);
+      serviceLocator.get('logger').info('cm-api response', response.body);
```
Comment (tomaszdurka) [inline lib/cm-api-client.js]: Same here?

File: lib/janus/connection.js [hunk 3]
```diff
@@ -60,7 +60,7 @@ JanusConnection.prototype.processMessage = function(message) {
   //This is an invalid case, so lets log all such messages
   var pluginId = message['sender'] || message['handle_id'] || null;
   if (pluginId) {
-    serviceLocator.get('logger').warn('Plugin message without session: ', message);
+    serviceLocator.get('logger').warn('Plugin message without session', message);
```
Comment (tomaszdurka) [inline lib/janus/connection.js]: Let's wrap message into `{request: message}`?

**Comment by vogdb** [comment]
@tomaszdurka please review

**Comment by tomaszdurka** [comment]
Fix plugin onRemove spec: `expect(logger.info.withArgs('Removed plugin', {plugin: plugin}).calledOnce).to.be.equal(true);`

**Comment by tomaszdurka** [comment]
Paired with @vogdb

**Pull request merged**